### ISSUE TITLE
Argument type mismatch

### ISSUE THREAD

**Issue by lvyandev** [Milad-Akarie/auto_route_library#213]
I have a constructor containing argument

```dart
import 'dart:ui' as ui;

final ui.Image image;

const TestPage({Key key, this.image,}) : super(key: key);
```

However, generared `router.gr.dart` recognizes it as `Image` from `widgets/image.dart`

```dart
import 'package:flutter/material.dart';

/// TestPage arguments holder class
class TestPageArguments {
  final Key key;
  final Image image;
  TestPageArguments({this.key, this.image,});
}
```

Thus, it compains

```dart
lib/app/router.gr.dart:296:23: Error: The argument type 'Image/*1*/' can't be assigned to the parameter type 'Image/*2*/'.
 - 'Image/*1*/' is from 'package:flutter/src/widgets/image.dart' ('../../flutter/packages/flutter/lib/src/widgets/image.dart').
 - 'Image/*2*/' is from 'dart:ui'.
          image: args.image,
```

**Comment by theweiweiway** [comment]
Is this still a problem in 1.0.1?

**Comment by Milad-Akarie** [comment]
@theweiweiway  That was actually fixed before 1.0.0
closing this.
